lib/nats.js

```diff
@@ -863,7 +863,7 @@ Client.prototype.processInbound = function() {
                         cb();
                     } // FIXME: Should we check for exceptions?
                 } else if ((m = PING.exec(buf)) !== null) {
-                    client.sendCommand(PONG_RESPONSE);
+                    client.sendCommand(PONG_RESPONSE);p
```

Comment:
@aricart did you mean to add this p?

Reply:
that was an error.  I pulled the request, out since the version was already bumped for 1.0.

Reply:
Ah right I see, cool. Thanks for doing the release!